doc/7/getting-started/.react-native/services/kuzzle.js

```diff
@@ -0,0 +1,10 @@
+import { Kuzzle, WebSocket } from 'kuzzle-sdk';
+
+const options = {
+    offlineMode: 'auto',
+    autoReplay: true,
```

Comment:
You don't need to set `autoReplay`, `autoQueue` and `autoResubscribe` when you set `offlineMode` to `auto` (the sdk do it for you)